nengo_viz/components/sim_control.py

```diff
@@ -93,9 +93,10 @@ def get_status(self):
 
     def javascript(self):
         info = dict(uid=self.uid)
-        json = self.javascript_config(info)
-        return 'sim = new VIZ.SimControl(control, %s); ' % json + \
-               'toolbar = new VIZ.Toolbar("%s"); ' % self.viz.viz.filename 
+        fn = json.dumps(self.viz.viz.filename)
+        js = self.javascript_config(info)
+        return ('sim = new VIZ.SimControl(control, %s);\n' \
```

Comment:
Again, the line continuation character `\` isn't necessary here :) There should be almost no places where it's necessary.